datacatalog/datacatalog_quickstart/main.go

```diff
@@ -0,0 +1,195 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START data_catalog_quickstart]
+
+// The datacatalog_quickstart application demonstrates how to define a tag
+// template, populate values in the template, and attach a tag based on the
+// template to a BigQuery table.
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	datacatalog "cloud.google.com/go/datacatalog/apiv1"
+	datacatalogpb "google.golang.org/genproto/googleapis/cloud/datacatalog/v1"
+)
+
+func main() {
+
+	var (
```

Comment:
Any particular reason to use a `var` group instead of `:=`?

Reply:
Mostly a carryover from idioms when they're globals.  Removed the var group.